tsl/src/fdw/deparse.c

```diff
@@ -327,6 +327,15 @@ contain_mutable_functions_checker(Oid func_id, void *context)
 	if (function_is_whitelisted(func_id))
 		return false;
 
+#ifndef NDEBUG
+	/* Special debug functions that we want to ship to data nodes. */
+	const char debug_func_prefix[] = "ts_debug_";
```

Comment:
Maybe the prefix should be something like `ts_debug_remote_` or `ts_debug_shippable_` since `ts_debug_` is quite generic and developers could easily create other debug functions with that prefix that they do not intend to get pushed-down to data nodes (causing failures if they haven't created the function on the data nodes).

Reply:
Will rename.